avm/utilities/pipelines/platform/Set-IssueForWorkflow.ps1

```diff
@@ -0,0 +1,128 @@
+<#
+.SYNOPSIS
+Check for failing pipelines and create issues for those, that are failing.
+
+.DESCRIPTION
+If a pipeline fails, a new issue will be created, with a link to the failed pipeline. If the issue is already existing, a comment will be added, if a new run failed (with the link for the new failed run). If a pipeline run succeeds and an issue is open for the failed run, it will be closed (and a link to the successful run is added to the issue).
+
+.PARAMETER Repo
+Mandatory. The name of the respository to scan. Needs to have the structure "<owner>/<repositioryName>"
+
+.PARAMETER LimitNumberOfRuns
+Optional. Number of recent runs to scan for failed runs. Default is 100.
+
+.PARAMETER LimitInDays
+Optional. Only runs in the past selected days will be analyzed. Default is 2.
+
+.PARAMETER IgnoreWorkflows
+Optional. List of workflow names that should be ignored (even if they fail, no ticket will be created). Default is an empty array.
+
+.EXAMPLE
+Set-IssueForWorkflow -Repo 'owner/repo01' -LimitNumberOfRuns 100 -LimitInDays 2 -IgnoreWorkflows @('Pipeline 01')
+
+Check the last 100 workflow runs in the repository 'owner/repo01' that happened in the last 2 days. If the workflow name is 'Pipeline 01', then ignore the workflow run.
+
+.NOTES
+The function requires GitHub CLI to be installed.
+#>
+function Set-IssueForWorkflow {
+  [CmdletBinding(SupportsShouldProcess, ConfirmImpact = 'Low')]
+  param (
+    [Parameter(Mandatory = $true)]
+    [string] $Repo,
+
+    [Parameter(Mandatory = $false)]
+    [int] $LimitNumberOfRuns = 100,
+
+    [Parameter(Mandatory = $false)]
+    [int] $LimitInDays = 2,
+
+    [Parameter(Mandatory = $false)]
+    [String[]] $IgnoreWorkflows = @()
+  )
+
+  $issues = gh issue list --state open --label 'automation,bug' --json 'title,url,body,comments' --repo $Repo | ConvertFrom-Json -Depth 100
```

Comment:
We should check with @matebarabas here which are the labels involved for gh issues managed via automation and related to failing workflows

Reply:
I changed for now to Type: AVM and Type: Bug

Reply:
@rahalan, please see (and only use) the approved labels that are available under the "AVM Standard GitHub Labels" dropdown here: https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr23---category-contributionsupport---github-repo-labels

If any additional labels are needed, let's discuss it internally! ;)

Reply:
The ones you picked ('Type: AVM :a: :v: :m:' and 'Type: Bug :bug:') seem to be a good start.